concrete/views/panels/page/versions.php

```diff
@@ -28,11 +24,11 @@
 			<% } %>
 			<div class="ccm-panel-page-versions-more-info">
 				<p><?=t('Edit by')?> <%-cvAuthorUserName%></p>
-				<% if (cvIsApproved == 1) { %>
+				<% if (cvIsApproved == 1 && cvApprovedDate && cvApproverUserName) { %>
```

Comment:
@marvinde What if we have only the date but not the author? In this case, I'd prefer to see the approval date...

Reply:
@mlocati You're right, we should display the informations for the following conditions :

- We have only the date
- We have only the author
- We have both

So I'll have to add the first 2 conditions. What do you think ?